source/includes/_themes_converting.md

```diff
@@ -0,0 +1,295 @@
+# Converting themes
+
+Earlier Mautic templates used PHP based on Symfony templates, however with the release of the 2.0 series, templates now use Twig.
```

Comment:
It would be great if we could provide explicit **information on whether it's mendatory** to convert PHP templates to Twig.
There's a case at my side where PHP templates remain functional after a core upgrade.
(See also: https://mautic.slack.com/archives/dev/p1471521162000270, CC: @escopecz)